examples/spread-in-binding-patterns/spread_in_binding_patterns.md

```diff
@@ -0,0 +1,7 @@
+# Rest binding pattern
+
+You can use the spread operator in mapping, list, and error binding patterns.
```

Comment:
```suggestion
You can use the rest binding pattern in mapping, list, and error binding patterns.
```

IMO, we need to improve the description to say what the rest binding pattern actually does. Similar to what we've specified for rest in match.